app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -133,4 +155,13 @@ def build_connect_params(options)
 
     connect_params
   end
+
+  def vm_host_handle(vm, options)
+    require 'Scvmm/miq_scvmm_vm_ssa_info'
+    raise "no credentials defined in options #{options}" if self.missing_credentials?(options[:auth_type])
+    user = options[:user] || authentication_userid(options[:auth_type])
+    pass = options[:pass] || authentication_password(options[:auth_type])
+
+    MiqScvmmVmSSAInfo.new(vm.host.hostname, user, pass)
```

Comment:
Does the current instance of `infra_manager` represent the VM's host or EMS? If it's the EMS, does the EMS's `user` and `pass` apply to the VM's host? If it's the host, should `vm.host.hostname` just be `hostname`?